modular-docs-manual/content/topics/module_creating-modules.adoc

```diff
@@ -0,0 +1,20 @@
+[[creating-modules]]
+= Creating Modules
+
+After identifying and creating the assemblies, each assembly should identify the modules that you need to create.
+
+.Procedure
+
+. For each assembly that you created, create a module file for each module that is identified in the assembly.
++
+Each assembly should already have the names of the modules that should be included in the assembly. Now you just need to create the actual files for those modules.
```

Comment:
Similar to my comment about "then add the names of the second-level assemblies".